### PR TITLE
added app.json file to ChangeUserRoles directory to list it on applis…

### PR DESCRIPTION
**Pull request by DravitLochan**
…t page

Fixes issue #283 
Changes: Added app.json file to the folder

### CODE REVIEW

**Comment by DravitLochan** [comment]
@saurabhjn76 @dynamitechetan @jyoth1raditya Please review!

File: html/apps/changeUserRoles/app.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "@context":"http://schema.org",
+  "@type":"SoftwareApplication",
+  "permissions":"/api/changeRoles",
+  "name":"changeuserroles",
+  "headline":"Change User Roles",
+  "alternativeHeadline":"Change User Roles",
+  "applicationCategory":"Accounts API",
+  "applicationSubCategory":"account operations",
+  "operatingSystem":"http://loklak.org",
```
Comment (dynamitechetan) [inline html/apps/changeUserRoles/app.json]: can this be changed to a SUSI URL.

Comment (DravitLochan) [inline html/apps/changeUserRoles/app.json]: Done!

**Comment by DravitLochan** [comment]
@jyoth1raditya @dynamitechetan @chashmeetsingh Please review!

**Comment by DravitLochan** [comment]
@mariobehling Please review and merge.

**Pull request merged**